Rakefile.dist

```diff
@@ -65,6 +66,14 @@ task :install => :build do |t|
 
   # ssh
   mkdir_p "#{DATAROOTDIR}/foreman/.ssh"
+
+  # cron scripts
+  mkdir_p "#{SYSCONFDIR}/cron.hourly"
+  cp_p "script/passenger-recycler.rb", "#{SYSCONFDIR}/cron.hourly/passenger-recycler"
+
+  # configuration files
+  mkdir_p "#{SYSCONFDIR}/foreman"
+  cp_p "script/passenger-recycler.rb.conf.example", "#{SYSCONFDIR}/foreman/passenger-recycler.rb.conf"
```

Comment:
The config file location here is defined by SYSCONFDIR, but the CONFIG path in the script is hardcoded, it should be set at build time.